ZenPacks/zenoss/HttpMonitor/configure.zcml

```diff
@@ -9,9 +9,9 @@
   <utility provides="zope.schema.interfaces.IVocabularyFactory"
            component=".info.httpMonitorRedirectVocabulary"
            name="httpMonitorRedirectVocabulary"/>
-  
+
   <subscriber zcml:condition="installed ZenPacks.zenoss.DistributedCollector.interfaces"
               for="ZenPacks.zenoss.DistributedCollector.DistributedPerformanceConf.DistributedPerformanceConf
                    ZenPacks.zenoss.DistributedCollector.interfaces.ICollectorInstalled"
-              handler=".onCollectorInstalled"/>
+              />
```

Comment:
You can remove the entire `subscriber` unit.  Without the `handler` attribute, it has no reason to exist.